### ISSUE TITLE
Missing capitalisation in notification emails for polls

### ISSUE THREAD

**Issue by bamstam** [loomio/loomio#4416]
Steps to reproduce, tested in German language version

1) create poll with case sensitive poll alternatives containing capitalisation (e. g. "Das ist eine mögliche Vorgehensweise")
2) receive notification emails
3) notification emails show poll alternatives with wrong capitalisation ("Das ist eine mögliche vorgehensweise") --> lower case only, whereas the first word is correctly capitalised

**Comment by robguthrie** [comment]
Hi Bamstam, thanks for the report. 

At first I want to know if this is actually your email client rather than Loomio.. I'm unaware of any places in the email pipeline where we do a text-transform like that.

Are you able to post the email message source in here, or forward the email to rob@loomio.org?

**Comment by bamstam** [comment]
Hi Rob, good point, i quickly double-checked with two other email clients, but all three show the notification email as described above:

* Thunderbird (52.4.0 32bit on Ubuntu Mate 14.04)
* Standard email client on Android 4.4.2
* Provider web interface at posteo.de

I will forward the email message source to your address in a minute.

**Comment by gdpelican** [comment]
Did you create this thread?

The only way I could think of this happening would be if the author created
a thread without capitalization, sent out email notifications, and then
updated the thread context to have the proper capitalization afterwards. In
any case we can have a more detailed look soon.

**Comment by bamstam** [comment]
@gdpelican Yes, I created the thread, poll alternatives were created with correct capitalisation and were not changed afterwards. Poll alternatives are correctly capitalised on Loomio, but not in the notification emails.
